bigbluebutton-config/web/index.html

```diff
@@ -282,4 +278,5 @@ <h3><a href="https://bigbluebutton.org/videos" title="More Videos" target="_blan
       </div>
     </footer>
   </body>
+  <script src="js/bigbluebutton.js"></script>
```

Comment:
this used to be imported in the `head`. Any reason to move it out?

Reply:
Same for `bigbluebutton-config/web/index_html5_vs_flash.html`

Reply:
Yes, I moved it to the bottom to have JavaScript code reached after the html, previously it was achieved by JQuery